src/components/Cluster/ClusterDetail/ClusterLabels/ClusterLabels.tsx

```diff
@@ -98,28 +98,29 @@ const ClusterLabels: FC<IClusterLabelsProps> = ({
       ) : (
         <>
           <LabelsWrapper>
-            {Object.entries(labels).map(([label, value]) => (
-              <LabelWrapper key={label}>
-                <EditLabelTooltip
-                  allowInteraction={!loading && allowEditing}
-                  label={label}
-                  onOpen={(isOpen) => setAllowEditing(isOpen)}
-                  onSave={save}
-                  value={value}
-                />
-                <DeleteLabelButton
-                  allowInteraction={!loading && allowEditing}
-                  onOpen={(isOpen) => setAllowEditing(isOpen)}
-                  onDelete={() => {
-                    save({ key: label, value: null });
-                  }}
-                  role='button'
-                  aria-label={`Delete '${label}' label`}
-                >
-                  &times;
-                </DeleteLabelButton>
-              </LabelWrapper>
-            ))}
+            {labels &&
```

Comment:
Narrows the type of `labels` and allows `Object.entries(labels).map` to pass the typechecker.